chromium_src/third_party/blink/public/mojom/permissions/permission.mojom

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+module blink.mojom;
+
+[BraveExtend]
+enum PermissionName {
+  BRAVE_ADS,
+  BRAVE_COSMETIC_FILTERING,
+  BRAVE_TRACKERS,
+  BRAVE_HTTP_UPGRADABLE_RESOURCES,
+  BRAVE_FINGERPRINTING_V2,
+  BRAVE_SHIELDS,
+  BRAVE_REFERRERS,
+  BRAVE_COOKIES,
+  BRAVE_SPEEDREADER,
+  BRAVE_ETHEREUM,
+  BRAVE_SOLANA,
+  BRAVE_GOOGLE_SIGN_IN,
+  BRAVE_LOCALHOST_ACCESS,
+  BRAVE_OPEN_AI_CHAT,
+};
```

Comment:
missing eol

Reply:
Fixed in https://github.com/brave/brave-core/pull/28471/commits/d12922308bc5f3c9dc2e8646020224fc0f1ad31a